src/components/Kite/story.tsx

```diff
@@ -0,0 +1,173 @@
+import Readme from './README.md';
+import { Wrap } from '../../stories/storybook-helpers';
+import Button from '../Button';
+
+function KiteExample() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const showKite = () => {
+    setIsOpen(true);
+  };
+
+  const hideKite = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <>
+      <div>
+        <Button modifiers={['primary']} onClick={showKite}>
+          Show Kite
+        </Button>
+      </div>
+      <Kite visible={isOpen} onClose={hideKite}>
+        <Kite.Header canBeDismissed title="The Title" onClose={hideKite}>
+          <Kite.Icon
+            fill="green"
+            height="20"
+            name="check"
+            style={{ display: 'block' }}
+            width="20"
+          />
+        </Kite.Header>
+      </Kite>
+    </>
+  );
+}
+
+function ErrorKiteExample() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const showKite = () => {
+    setIsOpen(true);
+  };
+
+  const hideKite = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <>
+      <div>
+        <Button modifiers={['primary']} onClick={showKite}>
+          Error Kite
+        </Button>
+      </div>
+      <Kite visible={isOpen} onClose={hideKite} ttl={3000}>
+        <Kite.Header canBeDismissed title="Error Kite" onClose={hideKite}>
+          <Kite.Icon
+            fill="red"
+            height="20"
+            name="warning"
+            style={{ display: 'block' }}
+            width="20"
+          />
+        </Kite.Header>
+      </Kite>
+    </>
+  );
+}
+
+function EverythingKiteExample() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const showKite = () => {
+    setIsOpen(true);
+  };
+
+  const hideKite = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <>
+      <div>
+        <Button modifiers={['primary']} onClick={showKite}>
+          Everything Kite
+        </Button>
+      </div>
+      <Kite visible={isOpen} onClose={hideKite}>
+        <Kite.Header
+          canBeDismissed
+          title="Kite with content"
+          onClose={hideKite}
+        >
+          <Kite.Icon
+            fill="red"
+            height="20"
+            name="warning"
+            style={{ display: 'block' }}
+            width="20"
+          />
+        </Kite.Header>
+        <Kite.Content>
+          <div className="t-flex">
+            <div>
+              <span>The quick brown fox jumps over the lazy dog!</span>
+            </div>
+            <div className="mt-2">
+              <Button modifiers={['primary']} onClick={() => {}}>
+                Click me!
+              </Button>
+            </div>
+          </div>
+        </Kite.Content>
+      </Kite>
+    </>
+  );
+}
+
+storiesOf('Galaxies/Kite', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Kite visible={false} ttl={3000} onClose={() => {}}>
+        <Kite.Header canBeDismissed title="Kite Title" onClose={() => {}}>
```

Comment:
Just curious: Does `Kite.Header` need its own `onClose` function? Wouldn't it be redundant to the function on `Kite`?

Reply:
Yeah, the `Kite.Header` need it's own `onClose` function because the `canBeDismissed` prop is set in `Kite.Header`. If it's set to true, we need the close button to call the onClose handler. I couldn't  figure out a way to pass the `onClose` function from the `Kite` to `Kite.Header`. If there's a better way to accomplish this, I'm all ears.

Reply:
I've done this with Context and by iterating over the children in the parent component and manually spreading props.
The context way is cleaner.

It might be better to do this sooner rather than later, so we don't end up with a bunch of `Kite.Header` components that have extra props. I think we can put both `canBeDismissed` and `onClose` as top-level `Kite` props.

Also, does the title need to be a string? If it's just getting rendered in the header, I think we should pass the header content as children and allow any React node as the type.

Reply:
For future readers:
Ended up moving `canBeDismissed` and `onClose` props to the `Kite` component, and nixing the context idea.